tvb_framework/tvb/interfaces/rest/bids_monitoring/bids_data_builder.py

```diff
@@ -0,0 +1,227 @@
+import os
```

Comment:
We can merge only if you agree to preserve our copyright header (like in other tvb-framework python files).

Reply:
Okay, i'll add the header